### DISCUSSION TITLE
v-treeview multiple item-children property value

### DISCUSSION THREAD

**Discussion by zaraemilien** [vuetifyjs/vuetify#14259] [Q&A]
Suppose i have variable items structured like below:

```
    [
      {
        id: 1,
        name: 'value1',
        item_nested: [
          {
            id: 2,
            name: 'value2',
          },
          {
            id: 3,
            name: 'value3',
            nested_item: [
              {
                id: 4,
                name: 'value4'
              }
            ]
          }
        ]
      }
    ]
```

In v-treeview we have props item-children that take string, it allow to set withc property will be taken as children reference. on the first level i want the property `item_nested` as children and the second level i want `nested_item` as children, but is it possible to set multiple value for item-children?

**Comment by ghost** [comment]
yes, taking a look at the documentation of [v-treeview](https://vuetifyjs.com/en/components/treeview/#color), you can see you can have multiple children nested with the following data:

```
items: [
  {
    id: 5,
    name: 'Documents',
    children: [
      {
        id: 6,
        name: 'vuetify',
        children: [
          {
            id: 7,
            name: 'src',
            children: [
              { id: 8, name: 'index : ts' },
              { id: 9, name: 'bootstrap : ts' },
            ],
          },
        ],
      }
    ],
  }
],
```

Now you can use this data in the `v-treeview` like so:
```
<v-treeview
    activatable
    color="warning"
    :items="items"
 ></v-treeview>
```
